src/tabs/style.ts

```diff
@@ -1,11 +1,24 @@
 import css from 'styled-jsx/css'
-import { color, space, font } from '_utils/branding'
+import { color, componentSizes, font, space } from '_utils/branding'
 
 const bottomBorderWidth = '2px'
 const bottomBorderOffset = '1px'
-const badgeOffset = '2px'
 
 export default css`
+  .kirk-tabs-wrapper {
+    position: relative;
+  }
+
+  .kirk-tabs-wrapper::before {
```

Comment:
Can't we use a normal border now that we this extra class? Or maybe we need an extra element again:
Main container could extend to 100%. And you could have a inner container with a max-width just for placing the tabs which would be the current `.wrapped` class

Reply:
That way we make sure we don't break the layout if tabs need to be in a column for example

Reply:
Alright, will try that way.

Reply:
Done, also fixed the selected tab border issue in scrollable mode so now borders look the same as in fixed.
<img width="258" alt="Screen Shot 2019-07-23 at 16 34 55" src="https://user-images.githubusercontent.com/373381/61722262-41960480-ad6a-11e9-895f-12de064ceb3f.png">